index.html

```diff
@@ -6,14 +6,232 @@
+                             Lorem ipsum dolor sit amet consectetur adipisicing elit. Quia veritatis, quod nam veniam sunt aperiam minus atque harum suscipit tempora non qui consequuntur esse repellendus et fugiat! Odit, fugit fugiat.
+                        </div>
+                    </div>
+                    <div class="about-col-6">
+                        <div class="image-card">
+                            <img src="./images/3.jpg" alt="cakes one">
+                        </div>
+                    </div> 
+                
+                    <div class="about-col-6">
+                        <div class="image-card">
+                            <img src="./images/1.jpg" alt="cakes one">
+                        </div>
+                    </div> 
+                    <div class="col-12 about-col-6">
+                        <div class="content">
+                            <h2>BUY ONLINE NOW &amp; GET 10% OFF!</h2>
+                            <button class="shop-now">Shop Now</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </div>
+    </dev>
+
+   <!--form section-->
+        <div class="container form-section">
+            <h3>For online orders, contact us</h3>
+            
+                    <form>
+                        <div class="row">    
+                           <div class="col-12 xl-col-6 first-name">
+                            <label>First Name</label><br>
+                            <input value="Yohannes" class="input-control" />
+                            </div>
+                            <div class="col-12 xl-col-6 last-name">
+                            <label class="right-inline">Last Name</label>
+                            <input value="Berhane" class="input-control" />
+                            </div>
+                        </div>
+                        <div class="row"> 
+                          <div class="col-12 xl-col-10 last-name">
+                            <label>Phone No</label>
+                            <input value="07412353196" class="input-control" />
+                          </div>
+                        
+                          <div class="col-12 xl-col-10 last-name">
+                            <label>Address</label>
+                            <input value="32 North End, NW12 3PQ" class="input-control" />                          
+                            <input class="submit-botton" type="submit" value="Send">
+                          </div>
+                        </div>
+                    
+                    </form>
+                           
+                </div>
+          
+    
+    <!--Footer-->
+    <div class="container section-footer">
```

Comment:
Use the `footer` tag here instead of a div, and move the footer outside of `main`